tools/webdriver/webdriver/client.py

```diff
@@ -431,6 +431,15 @@ def send_command(self, method, url, body=None):
 
         if "value" in response.body:
             value = response.body["value"]
+            """
+            Edge does not yet return the w3c session ID. 
+            We want the tests to run in Edge anyway to help with REC.
+            In order to run the tests in Edge, we need to hack around
+            bug:
+            https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/14641972
+            """
+            if url == "session" and method == "POST" and "sessionId" in response.body:
```

Comment:
Can we add an `and "sessionId" not in value` clause here, and log a warning in this case, please?

Reply:
@jgraham I added the additional clause, but I don't see other logging in this file yet. I don't mind adding it, but I'm not sure about the best way to do it.

Reply:
`logger.warning("Got legacy new session response")`

The logger is already imported even though it's not used.

Reply:
I should have mentioned that I tried logger.warning("foo") and logger.debug("foo") but both hit an error:
AttributeError: 'NoneType' object has no attribute 'warning'
Which I think means that the logger was not created successfully.